content/blog/nodepool-upgrades/index.md

```diff
@@ -0,0 +1,161 @@
+---
+path: '/nodepool-upgrades'
+title: 'Safer Node Pool Upgrades'
+date: 2022-03-28T06:00:00.000+00:00
+featured_image: hero-image.png
+categories: ['Pixie Team Blogs', 'Kubernetes']
+authors: ['Hannah Troisi']
+emails: ['htroisi@pixielabs.ai']
+---
+
+
+Are you dreading upgrading your cluster to a newer Kubernetes version? There are a few reasons why you might be motivated to upgrade. Perhaps you want to do one of the following:
+
+- Use a new beta API
+- Install an application that requires a more recent Kubernetes version
+- Follow the best practice of keeping your software up-to-date
```

Comment:
nit: up to date (I would use "up-to-date" when it is used more like a traditional adjective like "up-to-date software")